toolkit/tools/imageconfigvalidator/imageconfigvalidator.go

```diff
@@ -170,6 +175,11 @@ func validatePackages(config configuration.Config) (err error) {
 				return fmt.Errorf("%s: [SELinux] selected, but '%s' package is not included in the package lists", validateError, selinuxPkgName)
 			}
 		}
+		if len(systemConfig.Users) > 0 || len(systemConfig.Groups) > 0 {
+			if !foundUserAddPackage {
+				return fmt.Errorf("%s: add users require '%s' package that is not included in the package lists", validateError, userAddPkgName)
```

Comment:
Suggestion to make it clearer for the user what's missing:
```suggestion
				return fmt.Errorf("%s: the '%s' package must be included in the package lists when the image is configured to add users or groups", validateError, userAddPkgName)
```